src/complete.js

```diff
@@ -0,0 +1,35 @@
+const completedTodos = (e) => {
+  const checkbox = e.target;
+  let btnId = checkbox.id;
+  btnId = btnId.split('-');
+  const id = parseInt(btnId[1], 10);
+  const inputId = document.querySelector(`#activity-${id}`);
+  const existingTodos = JSON.parse(localStorage.getItem('todos'));
+  if (checkbox.checked) {
+    existingTodos[id].completed = true;
+    inputId.classList.add('completed');
+    localStorage.setItem('todos', JSON.stringify(existingTodos));
+  } else {
+    existingTodos[id].completed = false;
+    inputId.classList.remove('completed');
+    localStorage.setItem('todos', JSON.stringify(existingTodos));
```

Comment:
- Please consider storing the logic for storing data to the local storage in a function to make it more simple and reusable to avoid unnecessary repetitions on line 11, 15, and 32

Reply:
Thank you. Noted